### PR TITLE
UI Leader Board Updates, Model, route, API changes in Front-end

### PR DESCRIPTION
**Pull request by adityaevontech**
Leader board changes

### CODE REVIEW

File: frontend/migrations/versions/f524c4a47236_add_user_name_to_completed_challenge_table.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Create Date: 2022-01-17 13:08:53.794787
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'f524c4a47236'
+down_revision = '74f03c8be8dd'
```
Comment (gankurg) [inline frontend/migrations/versions/f524c4a47236_add_user_name_to_completed_challenge_table.py:14]: Why we are using the old revision numbers? @adityaevontech when you copy and paste the revions please change the revison numbers

Comment (adityaevontech) [reply frontend/migrations/versions/f524c4a47236_add_user_name_to_completed_challenge_table.py:14]: These revision numbers are generated by Alembic '74f03c8be8dd' is the old migration that was last performed.

Comment (adityaevontech) [reply frontend/migrations/versions/f524c4a47236_add_user_name_to_completed_challenge_table.py:14]: Kindly make/push docker_ui and docker_frontend tagged as latest

Comment (gankurg) [reply frontend/migrations/versions/f524c4a47236_add_user_name_to_completed_challenge_table.py:14]: why latest Aditya ?

**Review by gankurg — APPROVED** [review]

**Pull request merged**